app/views/expenses/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<section class="expenses-section">
+  <% if current_user %>
+        <%= render 'layouts/expenses_navbar' %>
+  <% end %>
+
+  <div class="d-flex gap-2 justify-content-between align-items-center border border-secondary-subtle w-100  px-2 mt-2">
+      <div>
+      <%= image_tag "#{@group.icon}", :class => 'img-fluid img-thumbnail w-50' %>
+      </div>
+      <div class="expense-group d-flex gap-2">
+    <p> <%="#{@group.name}: "%> </p>
+    <p style="font-weight:bold"><%= "$#{group_total(@group)}"%></p>
+    </div>
+  </div>
+
+  <div id="expenses" class="px-2">
+    <% @expenses.each do |expense| %>
```

Comment:
- [ ] Great job for adding the menu for navigation but it would be great if you added a bacl arrow as mentioned in the requirements to back to the home page when the user press on it since it is easier for the user so that would improve from user expereince 🤩💯🔥 so kindly add it and link it to the home page 💯


![image](https://user-images.githubusercontent.com/58553711/235318139-1c108eed-d774-4dc9-8fd4-1184bfd35401.png)